xml/http_servers.xml

```diff
@@ -4939,6 +4939,14 @@
     <param pos="0" name="hw.vendor" value="Server Technology"/>
   </fingerprint>
 
+  <fingerprint pattern="^frp/(\d+(?:\.\d+)*)$">
+    <description>frp - fast reverse proxy, not found response</description>
```

Comment:
It's interesting that the server header is only present on a 404 page 👀

Reply:
That was interesting (see [pkg/util/vhost/resource.go#L72](https://github.com/fatedier/frp/blob/2225a1781fc2774517b5d164c98675f09ba156e3/pkg/util/vhost/resource.go#L72))